src/test/java/sboot/example/CsvReaderUtilImplTest.java

```diff
@@ -1,55 +1,56 @@
 package sboot.example;
 
-import com.opencsv.CSVReader;
 import java.io.File;
 import java.io.FileNotFoundException;
-import java.io.FileReader;
 import java.util.List;
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.util.ResourceUtils;
 import sboot.example.util.CsvReaderUtil;
 import sboot.example.util.CsvReaderUtilImpl;
 
-
 public class CsvReaderUtilImplTest {
-    private static final List<String[]> CORRECT_LIST =
-            List.of(new String[]{"Id", "ProductId", "UserId", "ProfileName",
-                            "HelpfulnessNumerator", "HelpfulnessDenominator", "Score",
-                            "Time", "Summary", "Text"},
-                    new String[]{"1", "B001E4KFG0", "A3SGXH7AUHU8GW", "delmartian", "1",
-                            "1", "5", "1303862400",
-                            "Good Quality Dog Food", "I have bought several of the Vitality "
-                            + "canned dog food products and have found them all "
-                            + "to be of good quality. The product looks more like "
-                            + "a stew than a processed meat and it smells better. "
-                            + "My Labrador is finicky and she appreciates this product better than  most."});
+    private static final String PATH = "classpath:test_csv_reader.csv";
+    private CsvReaderUtil reader;
+
+    @BeforeEach
+    public void init() {
+        this.reader = new CsvReaderUtilImpl();
```

Comment:
don't think we need to new instance for each test, since we don't have any state that may change in this class